doi-proxy/src/main/java/no/unit/nva/doi/CrossRefClient.java

```diff
@@ -102,11 +123,11 @@ private HttpRequest createRequest(URI doiUri) {
                 .header(HttpHeaders.USER_AGENT, CROSSREF_USER_AGENT)
                 .timeout(Duration.ofSeconds(TIMEOUT_DURATION))
                 .GET();
-        if (getCrossRefApiPlusToken().isPresent()) {
-            logger.info(ADDING_TOKEN_IN_HEADER);
-            builder.setHeader(CROSSREF_PLUSAPI_HEADER,
-                    String.format(CROSSREF_PLUSAPI_AUTHORZATION_HEADER_BASE, getCrossRefApiPlusToken().get()));
-        }
+
```

Comment:
Before I refactor this, the previous functionality viewed misconfiguration of the secrets as OK and defaulted to the open Crossref API if no secret was found…do we still want this?